pom.xml

```diff
@@ -305,6 +305,11 @@
 			<version>${awaitility.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.yaml</groupId>
+			<artifactId>snakeyaml</artifactId>
+			<version>1.21</version>            
```

Comment:
the latest version is 1.25 https://mvnrepository.com/artifact/org.yaml/snakeyaml/1.25

Reply:
provided https://github.com/lhein/camel-language-server/pull/5

Reply:
done